pubspec.yaml

```diff
@@ -61,6 +62,9 @@ dev_dependencies:
   mockito: ^5.4.4
   ubuntu_lints: ^0.3.0
 
+dependency_overrides:
```

Comment:
Please add an explanation as to why adding this override.

Reply:
This was unnecessary. Removed

Reply:
Ok this was added because of a dependency issue that arises from dart code metrics. Here's the issue.

Because pub_updater <0.4.0 depends on process ^4.2.3 which depends on file ^6.0.0, pub_updater <0.4.0 requires file ^6.0.0.
And because every version of dart_code_metrics from git depends on file ^7.0.0, pub_updater <0.4.0 is incompatible with dart_code_metrics from git.
So, because keypod depends on dart_code_metrics from git which depends on pub_updater ^0.3.0, version solving failed.